components/raftstore/src/store/worker/read.rs

```diff
@@ -146,6 +146,47 @@ pub struct ReadDelegate {
     invalid: Arc<AtomicBool>,
     pub txn_extra_op: Arc<AtomicCell<TxnExtraOp>>,
     max_ts_sync_status: Arc<AtomicU64>,
+
+    track_ver: TrackVer,
+}
+
+#[derive(Debug)]
+struct TrackVer {
+    version: Arc<AtomicU64>,
+    local_ver: u64,
+    source: bool,
```

Comment:
What does `source` mean?

Reply:
It means the one created by `new` instead of `clone`, more specific, only the `ReadDelegate` created by `ReadDelegate::new` (store at `StoreMeta`) can increase version, because only these `ReadDelegate` will invoke `ReadDelegate::update`

Reply:
Can you also add it as comment?

Reply:
Done, PTAL again